tests/config_test.py

```diff
@@ -21,7 +22,11 @@
 from generic_config_updater.generic_updater import ConfigFormat
 
 import config.main as config
-import config.validated_config_db_connector as validated_config_db_connector
+imp.load_source(
```

Comment:
We do this change because a manual test run doesn't work. it fixes it.
By the manual I mean:
```
KEEP_SLAVE_ON=yes make <target>
### Wait ###
cd <target_source_dir>
python3 setup.py test
```
It is much faster to restart it like that, but it didn't work, so that change fixes it.

Reply:
@fastiuk I prefer not to have this change. All the tests run by default when executing build and should be able to run manually as well. Can you please check why it doesn't work only this specific file. All other imports work this way.